Kudu.Core/LinuxConsumption/NullMeshPersistentFileSystem.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Threading.Tasks;
+
+namespace Kudu.Core.LinuxConsumption
+{
+    public class NullMeshPersistentFileSystem : IMeshPersistentFileSystem
```

Comment:
@balag0 Let's a comment that this class is to avoid Null Checks, when running on non consumption SKU.

Reply:
Will do. thanks